docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -26,98 +26,176 @@ Sometimes we just need to reboot the jetson device when it gets stuck in initial
+```shell
+git clone https://github.com/pytorch/vision.git
 cd vision
-git co tags/v0.7.0 -b vision07
+git checkout v0.9.0
 pip install -e .
 ```
 
+**Note:** If the `pip install` throws error: `Illegal instruction (core dumped)`, then add `export OPENBLAS_CORETYPE=ARMV8` in `~/.bashrc` and then `source ~/.bashrc && conda activate mmdeploy` will solve the problem.
+
 ### Install mmcv
 
 Install openssl first:
+```shell
+sudo apt-get install -y libssl-dev
 ```
-sudo apt-get install libssl-dev
+Then install it from source, MMDeploy is using mmcv version is `1.4.0`:
+```shell
+git clone https://github.com/open-mmlab/mmcv.git
+cd mmcv
+git checkout v1.4.0
+MMCV_WITH_OPS=1 pip install -e .
 ```
-Then install it from source like `MMCV_WITH_OPS=1 pip install -e .`
 
 ### Update cmake
 
 We choose cmake version 20 as an example.
-```
+```shell
+sudo apt remove cmake
+sudo apt purge --auto-remove cmake
 sudo apt-get install -y libssl-dev
 wget https://github.com/Kitware/CMake/releases/download/v3.20.0/cmake-3.20.0.tar.gz
 tar -zxvf cmake-3.20.0.tar.gz
 cd cmake-3.20.0
 ./bootstrap
-make
+make -j$(nproc)
 sudo make install
 ```
 Then we can check the cmake version through:
-```
+```shell
 source ~/.bashrc
 cmake --version
 ```
 
-### Install mmdeploy
-Just follow the instruction [here](../build.md). If it throws `failed building wheel for numpy...ERROR: Failed to build one or more wheels` when installing `h5py`, try install `h5py` manually.
-```
-sudo apt-get install pkg-config libhdf5-100 libhdf5-dev
-pip install versioned-hdf5 --no-cache-dir
+### Install spdlog
+```shell
+sudo apt-get install libspdlog-dev
 ```
 
-Then install onnx manually. First, we have to install protobuf compiler:
-```
-sudo apt-get install libprotobuf-dev protobuf-compiler
-```
-Then install onnx through:
-```
+### Install onnx
+```shell
 pip install onnx
 ```
-Then reinstall mmdeploy.
 
+### Install pplcv
+A high-performance image processing library of openPPL. Now, MMDeploy supports `v0.6.2` and has to use git clone to download it.
+```shell
+git clone https://github.com/openppl-public/ppl.cv.git
+cd ppl.cv
+export PPLCV_DIR=$(pwd)
+git checkout tags/v0.6.2 -b v0.6.2
+./build.sh cuda
+```
+
+### Install h5py
+```shell
+sudo apt-get install pkg-config libhdf5-100 libhdf5-dev
+pip install versioned-hdf5
+```
+
+### Install MMDeploy
+Using git to clone MMDeploy source code.
+```shell
+git clone -b master https://github.com/open-mmlab/mmdeploy.git MMDeploy
+cd MMDeploy
+git submodule update --init --recursive
+````
+
+Build MMDeploy from source:
+```shell
+mkdir -p build && cd build
+cmake .. \
+    -DCMAKE_CXX_COMPILER=g++-7 \
+    -DMMDEPLOY_BUILD_SDK=ON \
+    -DMMDEPLOY_BUILD_SDK_PYTHON_API=ON \
+    -DMMDEPLOY_TARGET_DEVICES="cuda;cpu" \
+    -DMMDEPLOY_TARGET_BACKENDS="trt" \
+    -DMMDEPLOY_CODEBASES=all \
+    -Dpplcv_DIR=${PPLCV_DIR}/cuda-build/install/lib/cmake/ppl \
+    -DTENSORRT_DIR=/usr/src/tensorrt \
+    -DCUDNN_DIR=/etc/alternatives
```

Comment:
as I've forgotten the exact paths of tensorrt and cudnn, could you make sure they are right for `TENSORRT_DIR` and `CUDNN_DIR` on TX2 or Xavier?

Reply:
I only have Jetson Nano 😢 
So I modefied it to:

> ### Install MMDeploy
> Using git to clone MMDeploy source code.
> ```shell
> git clone -b master https://github.com/open-mmlab/mmdeploy.git MMDeploy
> cd MMDeploy
> git submodule update --init --recursive
> ````
> 
> We need the path of TensorRT and cuDNN path for MMDeploy installation.
> - for Jetson Nano:
> ```shell
> export TENSORRT_DIR=/usr/src/tensorrt
> export CUDNN_DIR=/etc/alternatives
> ```
> 
> Build MMDeploy from source:
> ```shell
> mkdir -p build && cd build
> cmake .. \
>     -DCMAKE_CXX_COMPILER=g++-7 \
>     -DMMDEPLOY_BUILD_SDK=ON \
>     -DMMDEPLOY_BUILD_SDK_PYTHON_API=ON \
>     -DMMDEPLOY_TARGET_DEVICES="cuda;cpu" \
>     -DMMDEPLOY_TARGET_BACKENDS="trt" \
>     -DMMDEPLOY_CODEBASES=all \
>     -Dpplcv_DIR=${PPLCV_DIR}/cuda-build/install/lib/cmake/ppl \
>     -DTENSORRT_DIR=${TENSORRT_DIR} \
>     -DCUDNN_DIR=${CUDNN_DIR}
> 
> make -j$(nproc) && make install
> ```

It is okay?

Reply:
@PeterH0323 Are you sure your TENSORRT_DIR is correct? On my Jetson, TensorRT is detected at:
```
-- Found TensorRT headers at /usr/include/aarch64-linux-gnu
-- Found TensorRT libs at /usr/lib/aarch64-linux-gnu/libnvinfer.so;/usr/lib/aarch64-linux-gnu/libnvinfer_plugin.so
```

I set TENSORRT_DIR to `/usr/include/aarch64-linux-gnu`

Reply:
Hi, @tehkillerbee. 
I am using Jetson Nano. May I ask what is the type of your Jetson?

Reply:
@PeterH0323 I have checked on both AGX Xavier and NX. /usr/src/tensorrt does exist - but it does not contain headers or libs as far as I can see. When you build MMDeploy, does it list where TensorRT headers, libs has been detected?

Reply:
@tehkillerbee  Done, the script show it can find trt when I set it to `/usr/src/tensorrt`, but thx for you reminding, I change both `-DTENSORRT_DIR` and `-DCUDNN_DIR` to `/usr/include/aarch64-linux-gnu` 😄